charts/manila-csi-plugin/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v1.32.0
 description: Manila CSI Chart for OpenStack
 name: openstack-manila-csi
-version: 2.33.0-alpha.0
+version: 2.33.0-alpha.1
```

Comment:
apologies, but the version just got bumped again: https://github.com/kubernetes/cloud-provider-openstack/blob/c6f2467cbfd3a48df017b74b7dedd5e3e927aef3/charts/manila-csi-plugin/Chart.yaml#L5


Can you set this to:  `2.33.0-alpha.2`